docs/reference-guides/interactivity-api/core-concepts/undestanding-global-state-local-context-and-derived-state.md

```diff
@@ -745,15 +745,15 @@ When using region-based navigation, it's crucial to ensure that your interactive
 
 `getServerState()` allows you to subscribe to changes in the **global state** that occur during client-side navigation. This function is analogous to `getServerContext()`, but it works with the global state instead of the local context.
 
+The `getServerState()` function returns a read-only reactive object. This means that any [callbacks](/docs/reference-guides/interactivity-api/api-reference.md#accessing-data-in-callbacks) you have defined that watch the returned object will only trigger when the value returned by the function changes. If the value remains the same, the callback will not re-trigger.
+
 Let's consider a quiz that has multiple questions. Each question is a separate page. When the user navigates to a new question, the server provides the new question and the time left to answer all the questions.
 
 ```php
-<?php
-wp_interactivity_state( 'myPlugin', array(
+<div <?php echo wp_interactivity_data_wp_context( array(
```

Comment:
You're explaining `getServerState`, so the example should use `wp_interactivity_state`, not `wp_interactivity_data_wp_context`, especially when later the JavaScript code refers to `state.question` not `context.question`.

Reply:
Right of course 🤦 .

Reply:
c78601fb7d1633a10cbbc6d5ddded607acb2d7bf